src/selectors/clusters.tsx

```diff
@@ -48,6 +48,7 @@ const clusterToClusterTableRow = (cluster: Cluster): IRow => {
         title: hostsCount.toString(),
         sortableValue: hostsCount,
       } as HumanizedSortable,
+      getRelativeTimeCell(createdAt),
```

Comment:
I think the user should be able to see even non-relative time. In a popover? @andybraren 

So far all attempts to show relative time throughout the application were reverted back to rendering real time. How is this different?